### ISSUE TITLE
Reference to baseurl in organization page missing trailing slash?

### ISSUE THREAD

**Issue by gaetawoo** [gohugoio/hugoDocs#122]
I know you need the final trailing slash in the baseurl, but this page:
https://github.com/gohugoio/hugoDocs/edit/master/content/content-management/organization.md

has 2 locations (lines 50 and 212) where it shows the baseurl _without_ the trailing slash. This ambiguity is very confusing to a new adopter.. what is the right answer here?

**Comment by digitalcraftsman** [comment]
Practically it doesn't matter whether the baseurl ends with a trailing slash or not. But I agree that it's better for beginners to showcase only one way to avoid confusion.

Hugo is clever enough to handle both types of baseurls. In case you've to construct urls inside a template then `absURL` and `relURL` are your best friends because they handle all the corner cases for you.

**Comment by gaetawoo** [comment]
@digitalcraftsman I agree, there should be one way described in the docs. My first site on Netlify wouldn't build correctly (well it built but all the references to css or other files were broken) because my baseurl didn't have the trailling slash. The theme probably didn't have the absURL or relURL for those references.

**Comment by felicianotech** [comment]
Looks like this issue has been taken care of?
